be/src/vec/functions/function_convert_tz.h

```diff
@@ -47,6 +51,10 @@ struct ConvertTZImpl {
                         const ColumnString* from_tz_column, const ColumnString* to_tz_column,
                         ReturnColumnType* result_column, NullMap& result_null_map,
                         size_t input_rows_count) {
+        auto convert_ctx = reinterpret_cast<ConvertTzCtx*>(
+                context->get_function_state(FunctionContext::FunctionStateScope::THREAD_LOCAL));
+        std::map<StringRef, cctz::time_zone> time_zone_cache_;
```

Comment:
```suggestion
        std::map<StringRef, cctz::time_zone> time_zone_cache;
```

Reply:
`time_zone_cache_` is a temporary map.
`time_zone_cache` is a reference to the map.